google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -1056,7 +1065,7 @@ private ResultSet executeQueryInternal(
         QueryOption... options) {
       Options readOptions = Options.fromQueryOptions(options);
       return executeQueryInternalWithOptions(
-          statement, queryMode, readOptions, null /*partitionToken*/);
+          statement, queryMode, readOptions, null /* partitionToken */);
```

Comment:
Please raise a separate PR for formatting changes.

Reply:
Removed all formatting changes from the PR.